R/descriptives.R

```diff
@@ -216,6 +216,27 @@ Descriptives <- function(jaspResults, dataset, options) {
     }
     .descriptivesScatterPlots(jaspResults[["scatterPlots"]], dataset.factors, variables, splitName, options)
   }
+  
+  # Dot plots
+  if(options[["descriptivesDotPlot"]]){
+    
+    if(is.null(jaspResults[["DotPlots"]])) {
+      jaspResults[["DotPlots"]] <- createJaspContainer(gettext("Dot Plots"))
+      jaspResults[["DotPlots"]]$dependOn(c("splitby", "descriptivesDotPlot"))
+      jaspResults[["DotPlots"]]$position <- 11
```

Comment:
The new stem and leaf table is already at position 11: https://github.com/jasp-stats/jaspDescriptives/blob/a51fd65d689417ce423960d2eeb01efef3ff2026/R/descriptives.R#L192